inspirehep/modules/workflows/views/callback.py

```diff
@@ -97,7 +148,13 @@ def webcoll_callback():
 
     Expects the request data to contain a list of record ids in the
     recids field.
+
+    DEPRECATED: replaced by the simpler continue_workflow_callback
```

Comment:
I know, we discussed this (sam and I) and though that simplicity is better as first step (webcoll runs each 5min, and there's a nice message saying that the record is there but will take a moment to show up)